lib/features/home/data/property_item_data.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:equatable/equatable.dart';
+import 'package:hatspace/data/property_data.dart';
+import 'package:intl/intl.dart';
+
+class PropertyItemData extends Equatable {
+  final String? id;
```

Comment:
Why is this nullable?

Reply:
@suesitran Because the id field in Property class is nullable

Reply:
should we update the Property class? id must be not null.

Reply:
Agree, ID must not be null